ffi-build.sh

```diff
@@ -64,7 +64,13 @@ actual_native_static_libs="$(cargo rustc --release --target ${target} -- --print
 echo "Actual native-static-libs:${actual_native_static_libs}"
 echo "Expected native-static-libs:${expected_native_static_libs}"
 
-[ "${expected_native_static_libs}" = "${actual_native_static_libs}" ]
+# compute libs in $actual_native_static_libs but not in $expected_native_static_libs
```

Comment:
[minor] The comment could say something along the lines of:
Comparing the unique elements between the expected and the actual native static libraries.

Reply:
Done